indy_node/utils/node_control_utils.py

```diff
@@ -45,8 +178,17 @@ def run_shell_command(cls, command, timeout=TIMEOUT):
     # Method is used in case we are NOT interested in command output
     # everything: command, errors, output etc are logged to journalctl
     @classmethod
-    def run_shell_script(cls, command, timeout=TIMEOUT):
-        subprocess.run(command, shell=True, timeout=timeout, check=True)
+    def run_shell_script(cls, command, stdout=False, stderr=False,
```

Comment:
Why did we change it? I think we have `run_shell_command`, and, `run_shell_script` should be used when when we are not interested in the output (see comments above)

Reply:
sometimes we expect both stdout/stderr and shell errors exceptions, neither `run_shell_command` nor `run_shell_script` provided such functionality. I will move new logic to separate API to keep these well tested ones.

Reply:
done